web/sites/default/settings.amezmo.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * @file
+ * Platform.sh settings.
+ */
+
+// Configure the database.
+if (isset($_ENV['APP_HOSTNAME'])) {
+  // TODO: FIND A WAY TO GET DB_HOST AND DB_USER
+  $databases['default']['default'] = [
+    'database' => $_ENV['DB_DATABASE'],
```

Comment:
For Drupal apps, `MYSQL_DATABASE`, `MYSQL_HOSTNAME`, `MYSQL_PASSWORD`, and `MYSQL_USER` can be used here which are generated in the .env file that Amezmo generates.